sympy/core/tests/test_eval_power.py

```diff
@@ -336,7 +334,7 @@ def test_issue_7638():
     assert sqrt(e**6) == e**3
     assert sqrt((1 + I*r)**6) != (1 + I*r)**3
 
-
+@XFAIL
```

Comment:
I was wondering that why were these `@XFAIL` added? I think these pass on master.